packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -467,6 +467,20 @@ const conf = (module.exports = convict({
     doc: 'The publically visible URL of the deployment',
     env: 'PUBLIC_URL',
   },
+  recovery_codes: {
+    count: {
+      default: 3,
```

Comment:
let's default this to the production value of 8 and set it in `server/config/local.json` as 3